src/backend/TrafficCourtsApi/ticket-worker/Features/Emails/EmailSender.cs

```diff
@@ -0,0 +1,145 @@
+﻿using FluentEmail.Core;
+using Gov.TicketWorker.Models;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using TrafficCourts.Common.Contract;
+
+namespace Gov.TicketWorker.Features.Emails
+{
+
+    public interface IEmailSender
+    {
+
+        void SendUsingTemplate(string to, string subject, TicketDisputeContract model);
+
+    }
+    public enum EmailTemplate
+    {
+        EmailConfirmation,
+        ChangeEmail
+    }
+
+    public class EmailSender : IEmailSender
+    {
+
+        private readonly IFluentEmail _email;
+        private readonly ILogger<EmailSender> _logger;
+
+        public EmailSender(IFluentEmail email, ILogger<EmailSender> logger)
+        {
+            _email = email;
+            _logger = logger;
+        }
+
+        private string dataURIScheme(string mimeType, string resource)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            Stream stream = assembly.GetManifestResourceStream(resource);
+            byte[] bytes;
+            MemoryStream memoryStream = new MemoryStream();
+            stream.CopyTo(memoryStream);
+            bytes = memoryStream.ToArray();
+
+            string base64Data = Convert.ToBase64String(bytes);
+            String dataScheme = String.Format("data:image/{0};base64,{1}==",
+                                mimeType,base64Data);
+            return dataScheme;
+            
+        }
+
+        public async void SendUsingTemplate(string to, string subject, TicketDisputeContract model)
+        {
+            DisputeEmail emailModel = constructDisputeEmailModel(model);
+            emailModel.LogoImage = dataURIScheme("png", "ticket-worker.Features.Emails.Resources.bc-gov-logo.png");
+            var result = await _email.To(to)
+            .Subject(subject)
+            .UsingTemplateFromEmbedded("ticket-worker.Features.Emails.Resources.submissiontemplate.liquid", emailModel, this.GetType().GetTypeInfo().Assembly)
+            .SendAsync();
+
+            if (!result.Successful)
+            {
+                _logger.LogError("Failed to send an email");
+            }
+        }
+
+        private DisputeEmail constructDisputeEmailModel(TicketDisputeContract model)
+        {
+            DisputeEmail disputeEmailModel = new DisputeEmail();
+                
+            disputeEmailModel.ConfirmationNumber = model.ConfirmationNumber;
+            disputeEmailModel.ViolationDate = model.ViolationDate;
+            disputeEmailModel.ViolationTicketNumber = model.ViolationTicketNumber;
+            disputeEmailModel.ViolationTime = model.ViolationTime;
+            disputeEmailModel.CountOneDescription = model.Offences[0].OffenceDescription;
```

Comment:
Are we guaranteed to have 3 offenses?  What if the `model.Offences` array has less than 3 items?